agent/agent.go

```diff
@@ -200,8 +200,6 @@ func handle(conn io.Writer, msg []byte) error {
 		fmt.Fprintf(conn, "num-gc: %v\n", s.NumGC)
 		fmt.Fprintf(conn, "enable-gc: %v\n", s.EnableGC)
 		fmt.Fprintf(conn, "debug-gc: %v\n", s.DebugGC)
-	case signal.Version:
```

Comment:
One more, we can know remove the signal.Version as well :)

Reply:
Yap, that's removed :)

Reply:
Removed `signal/Version` as well, PTAL.